lib/Twig/Extension/Core.php

```diff
@@ -537,26 +537,26 @@ function twig_urlencode_filter($url)
  *  {# items now contains { 'apple': 'fruit', 'orange': 'fruit', 'peugeot': 'car' } #}
  * </pre>
  *
- * @param array|Traversable $arr1 An array
- * @param array|Traversable $arr2 An array
+ * @param $arguments An array of array|Traversable which will be used to call array_merge.
```

Comment:
this documentation is wrong. The method signature does not have a `$arguments` argument (you have such variable internally, but the signature has several arguments instead)